src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs

```diff
@@ -95,6 +96,19 @@ public PreferenceSettings PreferenceSettings
             get { return Model.PreferenceSettings; }
         }
 
+        public PreferencesViewModel PreferencesViewModel
```

Comment:
Let's make it internal for now

Reply:
@QilongTang fixed to be internal in commit: https://github.com/RobertGlobant20/Dynamo/commit/b49c1903df29c06fb917c4cf0f2a2e5ba10e85eb